packages/app-project/src/screens/ClassifyPage/components/RecentSubjects/RecentSubjects.js

```diff
@@ -38,7 +36,7 @@ function RecentSubjects (props) {
               height={height}
               key={recent.subjectId}
               isLoggedIn={isLoggedIn}
-              placeholder={<img alt='' role='presentation' src='/static/subject-placeholder.png' />}
+              placeholder={<img alt='' role='presentation' src='/subject-placeholder.png' />}
```

Comment:
This will resolve to https://www.zooniverse.org/subject-placeholder.png, which I think will 404.

Reply:
it's been updated to use the `assetPrefix`